collector/processor/concurrentbatchprocessor/metrics.go

```diff
@@ -24,14 +21,11 @@ const (
 )
 
 var (
-	processorTagKey          = tag.MustNewKey("processor")
-	statBatchSizeTriggerSend = stats.Int64("batch_size_trigger_send", "Number of times the batch was sent due to a size trigger", stats.UnitDimensionless)
-	statTimeoutTriggerSend   = stats.Int64("timeout_trigger_send", "Number of times the batch was sent due to a timeout trigger", stats.UnitDimensionless)
-	statBatchSendSize        = stats.Int64("batch_send_size", "Number of units in the batch", stats.UnitDimensionless)
-	statBatchSendSizeBytes   = stats.Int64("batch_send_size_bytes", "Number of bytes in batch that was sent", stats.UnitBytes)
```

Comment:
Do we lose these metrics by removing OpenCensus?
If yes, how do we plan to reintegrate them?
If no, what is the rationale behind this?

Reply:
Thanks for the review! No this code is just leftover from forking the original batchprocessor.  We are using the otel equivalents [here](https://github.com/open-telemetry/otel-arrow/blob/main/collector/processor/concurrentbatchprocessor/metrics.go#L107-L110). OC vs Otel metrics is typically controlled by a featureflag, but for this component we are always setting useOtel to true, meaning these opencensus metrics never recorded values. i.e. I don't think we are losing anything besides an unneeded dependency.

Reply:
Cool. Thanks for the clarification.